docs/architecture/adr-036-arbitrary-signature.md

```diff
@@ -0,0 +1,103 @@
+# ADR 036: Arbitrary Message Signature Specification
+
+## Changelog
+
+- 28/10/2020 - Initial draft
+
+## Authors
+- Antoine Herzog (@antoineherzog)
+- Zaki Manian (@zmanian)
+- Aleksandr Bezobchuk (alexanderbez) [1]
+- Frojdi Dymylja (@fdymylja)
```

Comment:
Can you please add me to the Authors list if you accept my new format and justification?

`- Ethan Frey (@ethanfrey)`